packages/aragon-api/src/index.js

```diff
@@ -144,6 +168,22 @@ export class AppProxy {
         ).pipe(
           pluck('result')
         )
+      },
+      pastEvents: (options = {}) => {
+        const eventArgs = [
+          address,
+          jsonInterface.filter(
+            (item) => item.type === 'event'
+          ),
+          options
+        ]
+
+        return this.rpc.sendAndObserveResponse(
```

Comment:
Here and in `pastEvents()`, shouldn't this be a `sendAndObserveResponses()`, as `@aragon/wrapper` streams individual events (with the `from()`)?

Reply:
Well that depends on how it behaves when there are two nested `from` calls on a promise. `pastEvents` **only** returns a promise. In all the tests I've done so far this seemed fine. 


https://github.com/aragon/aragon.js/blob/4c22a1173f4d4cfeabba92014b03d1e0f6a25520/packages/aragon-wrapper/src/core/proxy/index.js#L41-L44

Reply:
Ahh gotcha, I see what's going on. I originally thought it was streaming the events one by one, but I think this makes more sense since it more closely aligns with the web3.js API anyway (all at once) 👍.